### PR TITLE
display string in pointer-documentation-pane

### PR DESCRIPTION
**Pull request by admich**
Old version of FRAME-DISPLAY-POINTER-DOCUMENTATION-STRING didn't
trigger the redisplay of the pointer-documentation-pane so the string
were not visible (e.g documentation in some scigraph commands).

----

#

### CODE REVIEW

File: Core/clim-core/frames.lisp [hunk 1]
```diff
@@ -1366,7 +1366,41 @@ have a `pointer-documentation-pane' as pointer documentation,
 
 (defun frame-display-pointer-documentation-string (frame string)
   (with-output-to-pointer-documentation (stream frame)
-    (write-string string stream)))
+      (write-string string stream))
+  (let ((*pointer-documentation-output* (frame-pointer-documentation-output frame)))
+    ;; To see the string it is necessary to trigger the redisplay of
+    ;; pointer-documentation-pane with FRAME-UPDATE-POINTER-DOCUMENTATION.
+    ;; I don't know why we need to trigger the update function twice to see
```
Comment (dkochmanski) [inline Core/clim-core/frames.lisp]: I think we need to first understand why it is needed to be called twice (probably that will reveal an issue somewhere else).

Comment (scymtym) [reply Core/clim-core/frames.lisp]: > I think we need to first understand why it is needed to be called twice (probably that will reveal an issue somewhere else).

I think I found a missing `force-output` around the pointer documentation code when working on the fb backend. Maybe that's the problem?

Comment (dkochmanski) [reply Core/clim-core/frames.lisp]: why not `finish-output`?

Comment (admich) [reply Core/clim-core/frames.lisp]: Actually the two calls of `FRAME-UPDATE-POINTER-DOCUMENTATION` are necessary only if you call `FRAME-DISPLAY-POINTER-DOCUMENTATION-STRING` from SLIME. If you call the function from a CLIM application one call is enough also if you send a string on a different application-frame. So we can:
1. continue to investigate why from `slime` two calls are necessary.
2. remove one of the two calls (`FRAME-DISPLAY-POINTER-DOCUMENTATION-STRING` is supposed to be called from CLIM not from SLIME).
3. leave the two calls

Comment (dkochmanski) [reply Core/clim-core/frames.lisp]: currently stream access from outside clim loop is prone to race conditions and it is a known problem, so I think it should be just a single call. there is a ticket for stream concurrent access so there is no need for a separate issue either.

**Comment by admich** [comment]
Removed the double call.

**Comment by admich** [comment]
I removed the double call of `FRAME-UPDATE-POINTER-DOCUMENTATION`. Are there other problems with this PR?

**Comment by nsrahmad** [comment]
@admich No problems on your side. I think @dkochmanski got busy with ECL stuff and forgot about this.

I don't see any more problem's so I am merging it.

**Pull request merged**